universum/github_handler.py

```diff
@@ -0,0 +1,61 @@
+import json
+import sys
+import requests
+
+from .modules.automation_server.jenkins_server import JenkinsServerForTrigger
+from .modules.vcs.github_vcs import GithubToken
+from .modules.output import needs_output
+from .modules.structure_handler import needs_structure
+from .lib.utils import make_block
+
+
+@needs_structure
+@needs_output
+class GithubHandler(JenkinsServerForTrigger, GithubToken):
+    """
+    Excepts payload from stdin!
+    """
+
+    @staticmethod
+    def define_arguments(argument_parser):
+        argument_parser.add_argument('--event', '-e', dest='event', metavar="GITHUB_EVENT",
+                                     help='Currently parsed from "x-github-event" header')
+        argument_parser.add_argument('--payload', '-p', dest='payload', metavar="PAYLOAD",
+                                     help='<actual help coming later> leave "-" to read from stdin')
+
+    def __init__(self, *args, **kwargs):
+        super(GithubHandler, self).__init__(*args, **kwargs)
+        if self.settings.payload == '-':
```

Comment:
This is just a reminder: we agreed to use curl-like approach:
- `-` to read from stdin
- `@` to read from file
- no special char to pass data directly

Reply:
Sure, this is just a blob